scripts/prepare.sh

```diff
@@ -79,14 +76,12 @@ pip3 install -r requirements.txt
 cd scripts
 if (( $build_sc_machine == 1 )); then
 	./make_all.sh
-	cat ../bin/config.ini >> ../../config/sc-web.ini
 fi
 cd ..
 
 
 prepare "sc-web"
 sudo yes | sudo pip3 install --default-timeout=100 future
```

Comment:
Actually, I'm not sure about this line, cause `sudo pip3 install` might cause a lot of troubles on some Linux distros, since it uses user directory (`~/.local/lib` in Arch for example). I think it would be better to replace with simple `pip3 install`

Reply:
I totally agree, but it's out of scope for this PR (it's not a great idea to add changes unrelated to the PR topic, because in case it'll get squashed and merged as a single commit, nobody will be able to meaningfully pick this fix without pulling all the other changes). Feel free to create a Trello card and make a PR to fix this though!